homeassistant/components/linkplay/media_player.py

```diff
@@ -0,0 +1,306 @@
+        """Send pause command."""
+        await self._bridge.player.pause()
+
+    @exception_wrap
+    async def async_media_play(self) -> None:
+        """Send play command."""
+        if self._bridge.player.status == PlayingStatus.PAUSED:
+            await self._bridge.player.resume()
+
+    @exception_wrap
+    async def async_set_repeat(self, repeat: RepeatMode) -> None:
+        """Set repeat mode."""
+        await self._bridge.player.set_loop_mode(REPEAT_MAP_INV[repeat])
+
+    @exception_wrap
+    async def async_play_media(
+        self, media_type: MediaType | str, media_id: str, **kwargs: Any
+    ) -> None:
+        """Play a piece of media."""
+        media = await media_source.async_resolve_media(
+            self.hass, media_id, self.entity_id
+        )
+        await self._bridge.player.play(media.url)
+
+    @exception_wrap
+    async def async_join_players(self, group_members: list[str]) -> None:
+        """Join `group_members` as a player group with the current player."""
+        multiroom = get_active_multiroom(self.hass, self._bridge)
+        if multiroom is None:
+            multiroom = LinkPlayMultiroom(self._bridge)
+
+        for group_member in group_members:
+            bridge = next(
+                (
+                    bridge
+                    for bridge in self._controller.bridges
+                    if bridge.device.uuid == group_member
+                ),
+                None,
+            )
+            await multiroom.add_follower(bridge)
+
+        await self._controller.discover_multirooms()
+
+    @exception_wrap
+    async def async_unjoin_player(self) -> None:
+        """Remove this player from any group."""
+        multiroom = get_active_multiroom(self.hass, self._bridge)
+        if multiroom is not None:
+            multiroom.remove_follower(self._bridge)
+
+    def _update_properties(self) -> None:
+        """Update the properties of the media player."""
+        self._attr_available = True
+        self._attr_state = STATE_MAP[self._bridge.player.status]
+        self._attr_volume_level = self._bridge.player.volume / 100
+        self._attr_is_volume_muted = self._bridge.player.muted
+        self._attr_repeat = REPEAT_MAP[self._bridge.player.loop_mode]
+        self._attr_shuffle = self._bridge.player.loop_mode == LoopMode.RANDOM_PLAYBACK
+        self._attr_sound_mode = EQUALIZER_MAP[self._bridge.player.equalizer_mode]
+        self._attr_supported_features = DEFAULT_FEATURES
+
+        if self._bridge.player.status == PlayingStatus.PLAYING:
+            if self._bridge.player.total_length != 0:
+                self._attr_supported_features = (
+                    self._attr_supported_features | SEEKABLE_FEATURES
+                )
+
+            self._attr_source = SOURCE_MAP.get(self._bridge.player.play_mode, "other")
+            self._attr_media_position = self._bridge.player.current_position / 1000
+            self._attr_media_position_updated_at = utcnow()
```

Comment:
How often does this fire? We don't want to have the state updated all the time just because the time is elapsing.

Reply:
I'm not quite sure what the default polling interval is through the entity's `async_update`, but it is updated through that